apps/system/js/rocketbar.js

```diff
@@ -502,6 +499,11 @@
       return true;
     },
 
+    _cancelSearch: function() {
```

Comment:
Because this is not actually canceling anything in progress, I would like to rename it to something like ```closeSearch``` instead.